packages/contracts/contracts/core/SphinxModuleProxyFactory.sol

```diff
@@ -34,14 +34,10 @@ contract SphinxModuleProxyFactory is ISphinxModuleProxyFactory {
     address private immutable MODULE_FACTORY = address(this);
 
     /**
-     * @notice Deploys the `SphinxModule` contract and initializes it so that nobody
-     *         can deploy directly through it.
+     * @notice Deploys the `SphinxModule` implementation contract via `CREATE2`.
      */
     constructor() {
         SphinxModule module = new SphinxModule{ salt: bytes32(0) }();
-        // We initialize the implementation using `address(1)` because its initializer checks that
-        // the Gnosis Safe address isn't `address(0)`.
-        module.initialize(address(1));
```

Comment:
We needed to remove this because `address(1)` isn't a valid Gnosis Safe proxy, which causes this call to fail.

Some background: the purpose of this line was to initialize the `SphinxModule` implementation contract. This prevents the possibility that an attacker could initialize the implementation with a malicious Safe proxy. We now enforce this in the [constructor of the `SphinxModule` itself](https://github.com/sphinx-labs/sphinx/pull/1241/files#diff-dc6de3ae0f4253bc2378e8c718b0096734adfa036fe5e57fa49f3902fba93b4aR111-R113). This is a better solution in general because this enforces that *any* `SphinxModule` implementation can't be initialized directly. Our previous solution just ensured that the `SphinxModule` implementation *deployed by the `SphinxModuleProxyFactory`* couldn't be initialized directly.